tools/clang/tools/libclang/dxcisenseimpl.h

```diff
@@ -362,6 +368,67 @@ class DxcType : public IDxcType
   HRESULT STDMETHODCALLTYPE GetKind(_Out_ DxcTypeKind* pResult) override;
 };
 
+class DxcCodeCompleteResults : public IDxcCodeCompleteResults
+{
+private:
+  DXC_MICROCOM_TM_REF_FIELDS()
+  CXCodeCompleteResults m_ccr;
```

Comment:
Must keep a pointer to results since the actual results are derived from this class.  This copy will not contain the desired results.  This is the cause of your crash during `~AllocatedCXCodeCompleteResults()`.